docs/how-to/craft/enable-classic-confinement.rst

```diff
@@ -3,18 +3,174 @@
+linter <reference-linters>` to check for common errors and problems in the snap's files.
+
+
+.. _how-to-enable-classic-confinment-example-project-files:
+
+Example project files with classic confinement
+----------------------------------------------
+
+The following projects are sample solutions for combining classic confinement with
+specific plugins. These are only guidelines. Every case is different, so you may need to
+make other adjustments to your snap's build to completely confine it.
+
+
+Autotools plugin
+~~~~~~~~~~~~~~~~
+
+`autotools-classic-example
+<https://github.com/snapcraft-doc-samples-unofficial/autotools-classic-example>`_ is a
+project with a main part that uses the Autotools plugin.
+
+.. collapse:: autotools-classic-example project file
+
+    .. literalinclude:: code/example-classic-confinement-autotools.yaml
+        :caption: snapcraft.yaml
+        :language: yaml
+        :lines: 2-
+
+
+Make plugin
+~~~~~~~~~~~
+
+`makefile-lib-example
+<https://github.com/snapcraft-doc-samples-unofficial/makefile-lib-example>`_ is a
+project with a main part that uses the Make plugin.
+
+.. collapse:: makefile-lib-example project file
+
+    .. literalinclude:: code/example-classic-confinement-make.yaml
+        :caption: snapcraft.yaml
+        :language: yaml
+        :lines: 2-
+
+
+CMake plugin
+~~~~~~~~~~~~
+
+`cmake-classic-example <https://github.com/snapcraft-docs/cmake-classic-example>`_ is a
+project with a main part that uses the CMake plugin.
+
+.. collapse:: cmake-classic-example project file
+
+    .. literalinclude:: code/example-classic-confinement-cmake.yaml
+        :caption: snapcraft.yaml
+        :language: yaml
+        :lines: 2-
+
+
+Python plugin
+~~~~~~~~~~~~~
+
+`python-ctypes-example <https://github.com/snapcraft-docs/python-ctypes-example>`_ is a
+project with a main part that uses the Python plugin.
+
+.. collapse:: python-ctypes-example project file
+
+    .. literalinclude:: code/example-classic-confinement-python.yaml
+        :caption: snapcraft.yaml
+        :language: yaml
+        :lines: 2-
+
+
+Patch Python ctypes to load system libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your app uses ctypes to access system libraries, it must be bundled with a patched
+version of the ctype files:
+
+1. Stage the system libraries in the main part. For the core22 base, the packages are:
+
+   .. code-block:: yaml
+       :caption: snapcraft.yaml
+
+       stage-packages:
+         - libpython3.10-minimal
+         - libpython3.10-stdlib
+
+2. Override the part's build to also run a shell script that patches the ctype files:
+
+   .. code-block:: yaml
+       :caption: snapcraft.yaml
+
+       override-build: |
+         snapcraftctl build
+         $SNAPCRAFT_PROJECT_DIR/snap/local/patch-ctypes.sh
+
+   The plugin now builds the part like normal before running the patch script.
+
+3. Build the snap.
+
+4. Build the snap. During build, the script looks for core Python modules that need to
```

Comment:
> core Python modules

This comes from the original doc. I'm not confident in these terms. Is there a more accurate phrase?